sspl_test/alerts/node/test_node_disk_actuator.py

```diff
@@ -27,7 +29,8 @@ def test_node_disk_module_actuator(agrs):
     check_sspl_ll_is_running()
     instance_id = "*"
     resource_type = "node:fru:disk"
-    send_node_controller_message_request(UUID, "NDHW:%s" % resource_type, instance_id)
+    target_node_id = Conf.get(SSPL_TEST_CONF, NODE_ID_KEY, DEFAULT_NODE_ID)
```

Comment:
build this target_node_id in common.py and feed that to send_node_controller_message_request() in same file, by default. that way you wont need changes in all this test code

Reply:
Done